hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2121,6 +2139,11 @@ func reconcileControlPlaneOperatorRole(role *rbacv1.Role) error {
 			Resources:     []string{"securitycontextconstraints"},
 			Verbs:         []string{"use"},
 		},
+		{
+			APIGroups: []string{"kubevirt.io"},
+			Resources: []string{"virtualmachines", "virtualmachineinstances"},
+			Verbs:     []string{rbacv1.VerbAll},
```

Comment:
hosted cluster shoudl only need view access to VM/VMIs

Reply:
The HostedControlPlane need to have thise permissions in order to grant them to the cloud-controller-manager controller
Will change it according to the cloud-controller-manager controller requirements